apps/mobile/src/queries/activity/account-activity.query.ts

```diff
@@ -6,6 +6,11 @@ import { QueryFunctionContext, useQuery } from '@tanstack/react-query';
 import { AccountAddresses, CryptoAsset } from '@leather.io/models';
 import { getActivityService } from '@leather.io/services';
 
+/**
```

Comment:
I added a lot of deprecations here as these aren't used. Maybe now is the time to remove? 

We still have the services for total everything but don't use the queries